Makefile

```diff
@@ -75,18 +76,14 @@ unittests:
 
 functional: build
 	@echo "Executing functional tests using built charm at ${PROJECTPATH}"
-	@CHARM_LOCATION=${PROJECTPATH} tox -e func -- ${FUNC_ARGS}
+	@TEST_JUJU_CHANNEL=2.9/stable CHARM_LOCATION=${PROJECTPATH} tox -e func -- ${FUNC_ARGS}
```

Comment:
I remember we tend to remove juju 2.9 controller in CI environment @rgildein , could you confirm this?

Reply:
Can we drop it in another pr? I would like to limit the scope of this PR.

Reply:
Okay to split it into 2 pr. 
But If juju 2.9 controller is not available, how could you pass the functional test?
We need to remove 2.9 first.

Reply:
This charm use github official runner for testing, so it should be fine.

Reply:
2.9 is gone, since only one project was using it, but this project is using LXD not our CI.